### ISSUE TITLE
npm-groovy-lint 6.0.0 install failed on Java 14

### ISSUE THREAD

**Issue by sinkcup** [nvuillam/npm-groovy-lint#77]
npm-groovy-lint 6.0.0 is using CodeNarc v1.6.1 which support Java 14.

but I install failed.

```
npm i npm-groovy-lint
grep groovy package-lock.json 
    "npm-groovy-lint": {
      "resolved": "https://.../npm-groovy-lint/download/npm-groovy-lint-6.0.0.tgz"

node ./node_modules/npm-groovy-lint/lib/index.js 
Java between 1.8 and 11.99 is required  (14.02 found)
Installing/Updating JRE in /Users/sinkcup/.java-caller...
Unexpected error: Command failed: npm install node-jre --save
npm WARN deprecated har-validator@5.1.5: this library is no longer supported
npm WARN java-caller-support@1.0.0 No description
npm WARN java-caller-support@1.0.0 No repository field.
npm WARN java-caller-support@1.0.0 No license field.

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! node-jre@0.2.3 install: `node install.js`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the node-jre@0.2.3 install script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

![image](https://user-images.githubusercontent.com/4971414/89246172-1876a800-d63d-11ea-80ef-0fcaf7d6380e.png)

**Comment by nvuillam** [comment]
I didn't notice, thanks for the tip ! 👍 
Now accepted java versions are from 8 to 14, please reopen the issue if you still see a problem with v6.1.0 🙂

**Comment by sinkcup** [comment]
Wow! It's OK now. Thx @nvuillam 
This is the most timely project I have ever seen!

**Comment by nvuillam** [comment]
You're welcome, it was really a tiny source change 🙃
